engine/menus/empty_sram.asm

```diff
@@ -1,12 +1,8 @@
 EmptyAllSRAMBanks:
-	ld a, 0
-	call .EmptyBank
-	ld a, 1
-	call .EmptyBank
-	ld a, 2
-	call .EmptyBank
-	ld a, 3
+for BANKNO, NUM_SRAM_BANKS
+	ld a, BANKNO
```

Comment:
`x` and `n` are common `for `indexes here (and it looks like we generally use `n` for 1-based and `x` for 0-based).
```suggestion
for x, NUM_SRAM_BANKS
	ld a, x
```